supplements/countKmers

```diff
@@ -1,61 +1,121 @@
-CORES=30 #number of cores to use for blast searches
-KMERSIZE=31 # RD:61
-
-#modified from NIKS script
-
-dir=/scratch/atif/1000_genomes/BEB		#directory for read files 
-hawkDir=/scratch/atif/hawk			#directory where hawk is installed
-jellyfishDir=/home/atif/jellyfish-Hawk/bin		#directory where jellyfish is installed
-sortDir=/home/atif/coreutils/deps/bin		#directory where parallel sort is installed
+#!/usr/bin/env bash
+set -eo pipefail
```

Comment:
It's helpful to halt execution if something goes wrong. Makes it easier to isolate the point of failure if it doesn't just try to keep going to the next sample.